operators/src/upgrade/common/constants.rs

```diff
@@ -44,3 +44,8 @@ pub(crate) const UPGRADE_OPERATOR_SERVICE: &str = "upgrade-operator-service";
 pub(crate) const UPGRADE_OPERATOR_SERVICE_PORT: i32 = 8080;
 /// Service internal port constant for upgrade operator.
 pub(crate) const UPGRADE_OPERATOR_INTERNAL_PORT: i32 = 8080;
+
+pub(crate) const NODE_LABEL: &str = "openebs.io/engine=mayastor";
+
+pub(crate) const DEFAULT_VALUES_PATH: &str =
```

Comment:
This should be a std::path::PathBuf. And then when using it in the command, you can use DEFAULT_VALUES_PATH.to_string_lossy().